samples/matter/smoke_co_alarm/sysbuild/mcuboot/boards/nrf54l15dk_nrf54l10_cpuapp.conf

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# TODO: Workaround Fprotect is not supported on nRF54l15 yet.
```

Comment:
```suggestion
# TODO: Workaround Fprotect is not supported on nRF54l10 yet.
```

Reply:
it should not even be here and in l15's overlay... god catch, thanks :)